File: 1012.c
/*
  Autor: Mateus Freitas;
  Nome: Área;
*/

#include <stdio.h>

int main(void){
    double a, b, c;
    double areaTri, areaCir, areaTrap, areaQuad, areaRet;

    scanf("%lf %lf %lf", &a, &b, &c);
    
    areaTri = (a * c) / 2;
    areaCir = (c * c) * 3.14159;
    areaTrap = ((a+b)*c) / 2;
    areaQuad = b * b;
    areaRet = a * b;
    printf ("TRIANGULO: %.3lf\n", areaTri);
    printf ("CIRCULO: %.3lf\n", areaCir);
    printf ("TRAPEZIO: %.3lf\n", areaTrap);
    printf ("QUADRADO: %.3lf\n", areaQuad);
    printf ("RETANGULO: %.3lf\n", areaRet);

    return 0;
}

File: Soluções Python/1012.py
#Autor: Mateus Freitas;
#Nome: Área;

a,b,c = list(map(float,input().split()))

area_triangulo = (a * c) / 2
area_circulo = 3.14159 * (c *c)
area_trapezio = ((a + b) * c) / 2
area_quadrado = b*b 
area_retangulo = a * b 

print("TRIANGULO: {:.3f}".format(area_triangulo))
print("CIRCULO: {:.3f}".format(area_circulo))
print("TRAPEZIO: {:.3f}".format(area_trapezio))
print("QUADRADO: {:.3f}".format(area_quadrado))
print("RETANGULO: {:.3f}".format(area_retangulo))
